packages/main/test/pages/Carousel.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width">
+
+	<title>Carousel</title>
+
+	<script src="../../webcomponentsjs/webcomponents-loader.js"></script>
+	<script src="../../resources/bundle.esm.js" type="module"></script>
+	<script nomodule src="../../resources/bundle.es5.js"></script>
+
+</head>
+
+<body style="background-color: var(--sapBackgroundColor); padding: 1rem;">
+<style>
+	ui5-carousel {
+		height: 20rem;
+		margin-bottom: 1rem;
+	}
+</style>
+<ui5-carousel id="carousel1" cycling="true">
+	<ui5-button>Button 1</ui5-button>
+	<ui5-button>Button 2</ui5-button>
+	<ui5-button>Button 3</ui5-button>
+</ui5-carousel>
+
+<ui5-carousel id="carousel2">
```

Comment:
If go to the 4th page (index 3) and set the attribute items-per-page="4", you will get 3 dots, none of them selected and a blank page.

3 dots, because there are 3 pages and the split is 4,4,2, which now is correct, but as the index 3 does not exist anymore, we get the blank page and none of the dots is selected - a bit strange state.

Maybe the component should recover from that state or if it`s too difficult we can leave it as less possible case.